Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -1093,7 +1095,25 @@ void FitPeaks::fitSpectrumPeaks(
 
     double expected_peak_pos = expected_peak_centers[peak_index];
 
-    // Determine how to set the starting value parameters
+    // clone peak function for each peak (need to do this so can
+    // set center and calc any parameters from xml)
+    auto peakfunction =
+        std::dynamic_pointer_cast<API::IPeakFunction>(m_peakFunction->clone());
+    peakfunction->setCentre(expected_peak_pos);
```

Comment:
If the _**observed**_ peak center is more accurate than the _**user-specified**_ peak center, which peak center will FitPeak use?

Reply:
Here it is preferable to use the knwon (user-specified) dSpacing to initialise the value of A and B instead of the observed peak centre (though one would hope they are very similar and the difference to A and B  would be negligible).